product_docs/docs/pem/8.0/pem_online_help/05_toc_pem_management_basics/05_storage_manager.mdx

```diff
@@ -36,12 +36,12 @@ Use the `Grid View` icon ![gridview](../images/gridview.png) to display all the
 
 Use the `Table View` icon ![tableview](../images/tableview.png) to display all the files and folders in a list view.
 
-Click on the check box next to *Show hidden files and folders* at the bottom of the window to view hidden files and folders.
```

Comment:
It will be good if we include these icons and descriptions in the form of a table.

Reply:
Rewritten in the table format as suggested.